pkg/core/interops.go

```diff
@@ -28,9 +28,6 @@ func SpawnVM(ic *interop.Context) *vm.VM {
 	vm := vm.NewWithTrigger(ic.Trigger)
 	vm.RegisterInteropGetter(getSystemInterop(ic))
 	vm.RegisterInteropGetter(getNeoInterop(ic))
-	if ic.Chain != nil {
-		vm.RegisterInteropGetter(ic.Chain.(*Blockchain).contracts.GetNativeInterop(ic))
```

Comment:
Are we dropping support for invoking native contracts via `System.Contract.Call` ?

Reply:
We shouldn't have `Neo.Native.Tokens.GAS` and `Neo.Native.Tokens.NEO` syscalls anymore, that's the whole point of this commit. Consequently, we don't need this `GetNativeInterop(...)` function. But we still should be able to invoke native contracts via `Neo.Native.Call` using contract name (e.g. `GAS`) and method. IIAC, this commit still keeps it possible.